### PR TITLE
ICU-21480 Promoted all @draft ICU 67 APIs to @stable ICU 67.

### PR DESCRIPTION
**Pull request by richgillam**
##### Checklist

- [x] Issue filed: https://unicode-org.atlassian.net/browse/ICU-21480
- [x] Updated PR title and link in previous line to include Issue number
- [x] Issue accepted
- [ ] Tests included
- [ ] Documentation is changed or added

### CODE REVIEW

**Review by markusicu — APPROVED** [review]
lgtm tnx

File: icu4c/source/i18n/unicode/measunit.h [hunk 1]
```diff
@@ -52,31 +52,32 @@ class LongNameHandler;
...
     /**
      * A mixed unit, like hour+minute.
      *
-     * @draft ICU 67
+     * @stable ICU 67
      */
     UMEASURE_UNIT_MIXED
 };
 
+#ifndef U_HIDE_DRAFT_API
```
Comment (markusicu) [inline icu4c/source/i18n/unicode/measunit.h:80]: looks like line 41 was un-removed and needs to be re-removed

Comment (richgillam) [reply icu4c/source/i18n/unicode/measunit.h:80]: Thank you; I'm an idiot.

Comment (richgillam) [reply icu4c/source/i18n/unicode/measunit.h:80]: Now I just need to figure out how to do that...

**Review by markusicu — COMMENTED** [review]
lgtm pse squash

**Review by markusicu — APPROVED** [review]
lgtm

**Review by sven-oly — COMMENTED** [review]
LGTM

**Pull request merged**